Lib/test/test_email/test_message.py

```diff
@@ -784,6 +784,80 @@ def test_str_defaults_to_utf8(self):
         m['Subject'] = 'unicöde'
         self.assertEqual(str(m), 'Subject: unicöde\n\n')
 
+    def test_folding_with_utf8_encoding_1(self):
+        # issue #36520
```

Comment:
It would be really helpful if these tests would add a one line comment about what are they actually testing. The name of the test uses numbers, so that doesn't help.

Something like,
`# Test header is folded correctly when maxlen falls in the middle of an encoded word.

Reply:
I added descriptive comments to the test cases. IMO, meaningfully descriptive test names would be too lengthy for these tests.